docs/fa/examples.md

```diff
@@ -0,0 +1,97 @@
+# Examples
+
+[FA Class](class-fa.md)  
+[NFA Class](class-nfa.md)  
```

Comment:
@eliotwrobson Apart from the fact that DFA should be listed before NFA to match the FA subclasses list, I'm not sure why we need these here. If we're gonna have them, we might as well have GNFA too. But per my reasoning in my other comment, I think of the top/bottom navigation as more for the purpose of hierarchy and 'Read Next'.

Reply:
sounds reasonable, I'll leave the table of contents link but remove the others.

Reply:
@eliotwrobson Sorry, should have been more specific: I would like to keep both FA and Table of Contents links here, since they are technically part of the hierarchy (I see the FA class page in this context as being the Table of Contents page for this `docs/fa` subdirectory).

In other words, I only want to remove the DFA and NFA links here.

Reply:
Done here https://github.com/caleb531/automata/pull/146/commits/2ce6816e437d8ca17b4e095ab4922e18eb4421f7